be/src/util/crc32c.cpp

```diff
@@ -225,8 +225,23 @@ uint32_t ExtendImpl(uint32_t crc, const char* buf, size_t size) {
     return static_cast<uint32_t>(l ^ 0xffffffffu);
 }
 
+uint32_t crc32c_sse42_simd(uint32_t crc, const char* buf, size_t len);
```

Comment:
consider declare this under conditional compile, keep consistent with its real impl in `crc32c_sse_simd.cpp`